CHANGES.rst

```diff
@@ -10,6 +10,13 @@ Changes
 
 - Cleaned up useless 2to3 conversion.
 
+- Introduce optionflag ``EXCEPTION_2TO3`` to normalize exception class names
```

Comment:
This has now been renamed, yes?

Also, I think "option flag" as two words works better.